perl-lib/OESS/lib/OESS/Database.pm

```diff
@@ -10330,6 +10330,38 @@ sub move_edge_interface_circuits {
             $self->_rollback() if($do_commit);
             return;
         }
+    
+        # Update new interface's cloud interconnect settings
+        $query = "UPDATE interface ".
```

Comment:
I see what you are trying to do here... but I think there are a few issues.  First its kinda hard to read / wrap head around.  I think one issue here is that there is a (select * from interface) which is unconstrained... so we have to do a full table scan.. then later on applying a where to that table scan.  

I think its actually easier to break this into 2 queries...

select * from interface where interface_id = ? (original interface_id)

then update interface set ..... 

and use the details of the first query to fill in the second...

Reply:
Updated. Let me know what you think